lib/Service.php

```diff
@@ -230,6 +230,33 @@ function mapValueObject($elementName, $className) {
         $this->classMap[$className] = function(Writer $writer, $valueObject) use ($namespace) {
             return \Sabre\Xml\Serializer\valueObject($writer, $valueObject, $namespace);
         };
+        $this->valueObjectMap[$className] = $elementName;
```

Comment:
Sorry I'm flip-flopping again. I think we need to just let people overwrite these if they want to